webapp/src/components/pages/TaskList.vue

```diff
@@ -89,6 +99,27 @@ const sortedTasks = computed(() => [...tasks.value.values()].sort(
   (task1, task2) => task1.displayInformation.taskTitle.localeCompare(task2.displayInformation.taskTitle)
 ))
 
+function getSchemeColor(task: Task) {
```

Comment:
hoo, nice idea to vary color based on task type!

consider returning `string` has to ensure that's it won't be undef when updating `taks.trainingInformation.scheme` (same remark for `getDataTypeColor`)

```suggestion
function getSchemeColor(task: Task): string {
```